rpcs3/Emu/Cell/Modules/sysPrxForUser.h

```diff
@@ -42,6 +42,19 @@ struct sys_lwmutex_locker
 	}
 };
 
+enum
+{
+	SYS_CRASH_DUMP_MAX_LABEL_SIZE = 16,
+	SYS_CRASH_DUMP_MAX_LOG_AREA   = 127 // not actually defined in CELL
+};
+
+struct sys_crash_dump_log_area_info_t
+{
+	char label[SYS_CRASH_DUMP_MAX_LABEL_SIZE]; // 15 + 1 (0 terminated)
+	vm::ptr<void> addr{ vm::null };
+	be_t<u64> size{ 0 };
```

Comment:
Why are there default initializers here? its not like we are going to construct objects from this struct.

Reply:
True, you can always zero it by empty aggregate initialization.

Reply:
Is it actually a problem or just a nitpick? :t

Reply:
A nitpick, but its better to change it.

Reply:
Okay i'll change it later